From f4d64d55ea825ec0d05aaa76d58cddc37d44f719 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Sat, 18 Jul 2015 13:02:25 +0200
Subject: [PATCH] dmsquash-generator.sh: increase timeout for checkisomd5

if the checkisomd5 takes longer than 3 minutes, the device timeout for
/dev/mapper/live-rw cancels the boot process.
---
 modules.d/90dmsquash-live/dmsquash-generator.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/modules.d/90dmsquash-live/dmsquash-generator.sh b/modules.d/90dmsquash-live/dmsquash-generator.sh
index e386d81..85a3592 100755
--- a/modules.d/90dmsquash-live/dmsquash-generator.sh
+++ b/modules.d/90dmsquash-live/dmsquash-generator.sh
@@ -64,3 +64,9 @@ ROOTFLAGS="$(getarg rootflags)"
     echo "What=/dev/mapper/live-rw"
     [ -n "$ROOTFLAGS" ] && echo "Options=${ROOTFLAGS}"
 } > "$GENERATOR_DIR"/sysroot.mount
+
+mkdir -p "$GENERATOR_DIR/dev-mapper-live\x2drw.device.d"
+{
+    echo "[Unit]"
+    echo "JobTimeoutSec=3000"
+} > "$GENERATOR_DIR/dev-mapper-live\x2drw.device.d/timeout.conf"
